<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Kirans-MacBook-Air.local" name="Test" tests="4" failures="1" timestamp="27 Sep 2016 11:22:15 GMT" time="12.86" errors="0">
  <testcase name="InsertMarketPlaceOrderTest" time="2.426" classname="apiTests.PostInsertMarketPlaceOrder"/>
  <testcase name="CrmAssignmentTest" time="3.227" classname="apiTests.PostCrmAssignment">
    <failure type="junit.framework.AssertionFailedError" message="
Expected: null
  Actual: CrmAssignmentResponse&amp;lt;message=&amp;quot;crm updated successfully for order Auto%d&amp;quot;, success=true&amp;gt;

--- Found following differences ---
 expected: null
   actual: CrmAssignmentResponse&amp;lt;message=&amp;quot;crm updated successfully for order Auto%d&amp;quot;, success=true&amp;gt;

">
      <![CDATA[junit.framework.AssertionFailedError: 
Expected: null
  Actual: CrmAssignmentResponse<message="crm updated successfully for order Auto%d", success=true>

--- Found following differences ---
 expected: null
   actual: CrmAssignmentResponse<message="crm updated successfully for order Auto%d", success=true>


	at junit.framework.Assert.fail(Assert.java:50)
	at apiTests.PostCrmAssignment.CrmAssignmentTest(PostCrmAssignment.java:66)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- CrmAssignmentTest -->
  <testcase name="FeAssignmentTest" time="4.638" classname="apiTests.PostFeAssignment"/>
  <testcase name="CancelOrderTest" time="1.937" classname="apiTests.PostCancelOrder"/>
</testsuite> <!-- Test -->
